Features/Intel/UserInterface/VirtualKeyboardFeaturePkg/VirtualKeyboardDxe/Keyboard.c

```diff
@@ -881,35 +881,45 @@ VkApiStart (
     goto Error;
   }
 
-  KeyData.KeyState.KeyToggleState = 0;

```

Comment:
Please keep these 0 or NULL initializations as it is for KeyToggleState, KeyShiftState, UnicodeChar and NotifyHandle

Reply:
I agree that the KeyData init and NotifyHandle init should not be removed.